ci/aws-lambda-ruby-dev/3.2/Dockerfile

```diff
@@ -1,19 +1,24 @@
 FROM public.ecr.aws/lambda/ruby:3.2
 
-# Install tools used when installing / building gems
-RUN yum install -y postgresql-devel sqlite-devel openssl-devel wget make gcc gcc-c++ git \
- && yum clean all \
- && rm -rf /var/cache/yum
```

Comment:
We included these (within an individual step) to minimize image size after each yum step.

Reply:
Still missing the clean up for a smaller image?

Reply:
🤦 Added now